app/src/main/java/de/danoeh/antennapod/activity/DownloadAuthenticationActivity.java

```diff
@@ -73,31 +72,25 @@ protected void onCreate(Bundle savedInstanceState) {
 
         txtvDescription.setText(txtvDescription.getText() + ":\n\n" + request.getTitle());
 
-        butCancel.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                setResult(Activity.RESULT_CANCELED);
-                finish();
-            }
+        butCancel.setOnClickListener(v -> {
+            setResult(Activity.RESULT_CANCELED);
+            finish();
         });
 
```

Comment:
unfortunately, android studio also changed the indentation with the new lambda expressions, so you'll be seeing many more lines marked as changed than it was supposed.